lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_firebase_test.rb

```diff
@@ -0,0 +1,183 @@
+
+              UI.user_error!("Invalid application APK: #{value}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_apk_path,
+            description: 'The test APK',
+            type: String,
+            verify_block: proc do |value|
+              next if File.file? value
+
+              UI.user_error!("Invalid test APK: #{value}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :model,
+            description: 'The device model to run',
+            type: String,
+            verify_block: proc do |value|
+              model_names = Fastlane::FirebaseDevice.valid_model_names
+              next if model_names.include? value
+
+              UI.user_error!("Invalid Model Name: #{value}. Valid Model Names: #{model_names}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :version,
+            description: 'The device version to run',
+            type: Integer,
+            verify_block: proc do |value|
+              version_numbers = Fastlane::FirebaseDevice.valid_version_numbers
+              next if version_numbers.include? value
+
+              UI.user_error!("Invalid Version Number: #{value}. Valid Verison Numbers: #{version_numbers}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :locale,
+            description: 'The locale code to run in',
+            type: String,
+            default_value: 'en',
+            verify_block: proc do |value|
+              locale_codes = Fastlane::FirebaseDevice.valid_locales
+              next if locale_codes.include? value
+
+              UI.user_error!("Invalid Locale: #{value}. Valid Locales: #{locale_codes}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :orientation,
+            description: 'Which orientation to run the device in',
+            type: String,
+            default_value: 'portrait',
+            verify_block: proc do |value|
+              orientations = Fastlane::FirebaseDevice.valid_orientations
+              next if orientations.include? value
+
+              UI.user_error!("Invalid Orientation: #{value}. Valid Orientations: #{orientations}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :type,
+            description: 'Which type of test are we running?',
+            type: String,
+            default_value: 'instrumentation',
+            verify_block: proc do |value|
+              types = Fastlane::FirebaseTestRunner::VALID_TEST_TYPES
+              next if types.include? value
+
+              UI.user_error!("Invalid Test Type: #{value}. Valid Types: #{types}")
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_run_id,
+            description: 'A unique ID used to identify this test run',
+            type: String,
+            default_value: run_uuid,
```

Comment:
Tbh this is not super idiomatic to have a dynamic value set as `default_value` here, because that means that each time you run `fastlane action android_firebase_test` to get the documentation about the action, you'll get a different UUID shown in the documentation for the default value.

I'd suggest instead to remove `default_value: run_uuid` here, as well as the one in `results_output_dir`, and as well as the `run_uuid = ` at the top of this `self.available_options`, and instead move that logic inside `self.run`

Reply:
Addressed in bd7395e4e97c05213a83f30d74d68b10b841a853 and tested [here](https://buildkite.com/automattic/woocommerce-android/builds/3359#40236a2f-b397-4f00-a87f-e67a1079769e) to ensure that everything works fine with no adjustment to the calling conventions.